src/js/tech/html5.js

```diff
@@ -549,9 +549,19 @@ class Html5 extends Tech {
    * @method setSrc
    */
   setSrc(src) {
+    let loadstartlistener = Html5.prototype.loadStartListener_;
```

Comment:
Are we explicitly using the `Html5#loadStartListener_` function instead of `this.loadStartListener_` because we want to be sure that property hasn't been added to an instance of the tech as something else? Maybe this is a candidate for being module-level private instead of class-level pseudo-private?